android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -388,24 +382,30 @@ private void showAnalyticsConsentPage() {
             mBtnNegative.setVisibility(View.VISIBLE);
         }
 
+        // Handle crash reporting consent based on installation status
         if (PackageUtils.isFirstInstall(this)
                 && !OnboardingPrefManager.getInstance().isP3aCrashReportingMessageShown()) {
+            // For first time installs, enable crash reporting by default
             if (mCheckboxCrash != null) {
                 mCheckboxCrash.setChecked(true);
             }
+            // Update metrics reporting consent
             UmaSessionStats.changeMetricsReportingConsent(
                     true, ChangeMetricsReportingStateCalledFrom.UI_FIRST_RUN);
+            // Mark crash reporting message as shown
```

Comment:
I'd remove this comment because it says the same thing as the following method invocation.